osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/k8s/Label.java

```diff
@@ -29,15 +29,23 @@
 import javax.persistence.Table;
 
 import org.osc.core.broker.model.entities.BaseEntity;
+import org.osc.core.broker.model.entities.virtualization.ProtectionEntity;
 import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMemberType;
 
 @SuppressWarnings("serial")
 @Entity
 @Table(name = "LABEL")
-public class Label extends BaseEntity {
-    @Column(name = "value", nullable = false)
+public class Label extends BaseEntity implements ProtectionEntity {
+    public Label() {
+    }
+
+    @Column(name = "value", nullable = false, unique = true)
```

Comment:
name and value may not be unique. The label/port and other entities are shared across vc's